text_extensions_for_pandas/io/test_watson.py

```diff
@@ -0,0 +1,879 @@
+        "text": "Now Santa Claus is missing.",
+        "location": [
+          69,
+          96
+        ]
+      },
+      {
+        "text": "-- Steven Wright",
+        "location": [
+          97,
+          113
+        ]
+      }
+    ]
+  },
+  "semantic_roles": [
+    {
+      "subject": {
+        "text": "The Bermuda Triangle"
+      },
+      "sentence": "The Bermuda Triangle got tired of warm weather.",
+      "object": {
+        "text": "of warm weather"
+      },
+      "action": {
+        "verb": {
+          "text": "tire",
+          "tense": "past"
+        },
+        "text": "tired",
+        "normalized": "tire"
+      }
+    },
+    {
+      "subject": {
+        "text": "Santa Claus"
+      },
+      "sentence": " Now Santa Claus is missing.",
+      "action": {
+        "verb": {
+          "text": "miss",
+          "tense": "present"
+        },
+        "text": "is missing",
+        "normalized": "be miss"
+      }
+    }
+  ],
+  "relations": [],
+  "language": "en",
+  "keywords": [
+    {
+      "text": "Bermuda Triangle",
+      "sentiment": {
+        "score": -0.866897,
+        "label": "negative"
+      },
+      "relevance": 0.998991,
+      "emotion": {
+        "sadness": 0.461508,
+        "joy": 0.245015,
+        "fear": 0.06578,
+        "disgust": 0.044216,
+        "anger": 0.248536
+      },
+      "count": 1
+    },
+    {
+      "text": "Santa Claus",
+      "sentiment": {
+        "score": -0.940095,
+        "label": "negative"
+      },
+      "relevance": 0.977698,
+      "emotion": {
+        "sadness": 0.638228,
+        "joy": 0.047636,
+        "fear": 0.217273,
+        "disgust": 0.015876,
+        "anger": 0.269789
+      },
+      "count": 1
+    },
+    {
+      "text": "warm weather",
+      "sentiment": {
+        "score": -0.866897,
+        "label": "negative"
+      },
+      "relevance": 0.878971,
+      "emotion": {
+        "sadness": 0.461508,
+        "joy": 0.245015,
+        "fear": 0.06578,
+        "disgust": 0.044216,
+        "anger": 0.248536
+      },
+      "count": 1
+    }
+  ],
+  "entities": [
+    {
+      "type": "Location",
+      "text": "Alaska",
+      "sentiment": {
+        "score": -0.940095,
+        "label": "negative"
+      },
+      "relevance": 0.963296,
+      "count": 1,
+      "confidence": 0.999498
+    },
+    {
+      "type": "Person",
+      "text": "Steven Wright",
+      "sentiment": {
+        "score": 0,
+        "label": "neutral"
+      },
+      "relevance": 0.252317,
+      "count": 1,
+      "confidence": 0.998251
+    }
+  ]
+}
+
+
+class TestWatson(unittest.TestCase):
+
+    def setUp(self):
+        # Ensure that diffs are consistent
+        pd.set_option("display.max_columns", 250)
+
+    def tearDown(self):
+        pd.reset_option("display.max_columns")
+
+    def test_print_all(self):
+        result = watson_nlp_parse_response(response1)
+        for k, v in result.items():
+            print("\n\n{}:".format(k))
```

Comment:
This output should go to a buffer so that it doesn't hide the console output of `unittest` when running regression tests.